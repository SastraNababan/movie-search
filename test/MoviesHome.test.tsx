import React from 'react'
import Movies from '../pages/movies'
import { render } from './testUtils'

describe('Movies Home page', () => {
  test('smoke test', () => {
    render(<Movies />)
  })

  it('snapshot', () => {
    const { asFragment } = render(<Movies />)
    expect(asFragment()).toMatchSnapshot()
  })
})
