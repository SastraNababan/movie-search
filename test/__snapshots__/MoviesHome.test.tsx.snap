// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movies Home page snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="css-0"
    >
      <div
        class="css-15888rv"
      >
        <div
          class="css-zbg2gu"
        >
          <h2
            class="chakra-heading css-f6t6fr"
          >
            Movie Search
          </h2>
        </div>
        <div
          class="css-17xejub"
        />
        <button
          aria-label="Switch to dark mode"
          class="chakra-button css-1t0t7oo"
          type="button"
        >
          <svg
            aria-hidden="true"
            fill="currentColor"
            focusable="false"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M283.211 512c78.962 0 151.079-35.925 198.857-94.792 7.068-8.708-.639-21.43-11.562-19.35-124.203 23.654-238.262-71.576-238.262-196.954 0-72.222 38.662-138.635 101.498-174.394 9.686-5.512 7.25-20.197-3.756-22.23A258.156 258.156 0 0 0 283.211 0c-141.309 0-256 114.511-256 256 0 141.309 114.511 256 256 256z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      class="chakra-container css-ggdveb"
    >
      <div
        class="css-l9lmd0"
      >
        <form>
          <div
            class="css-79elbk"
          >
            <div
              class="chakra-stack css-84zodg"
            >
              <div
                class="chakra-input__group css-4302v8"
              >
                <div
                  class="chakra-input__left-element css-9murld"
                >
                  <svg
                    class="chakra-icon css-zgjiqu"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M23.384,21.619,16.855,15.09a9.284,9.284,0,1,0-1.768,1.768l6.529,6.529a1.266,1.266,0,0,0,1.768,0A1.251,1.251,0,0,0,23.384,21.619ZM2.75,9.5a6.75,6.75,0,1,1,6.75,6.75A6.758,6.758,0,0,1,2.75,9.5Z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
                <input
                  class="chakra-input css-13h0e6n"
                  placeholder="Search movies ..."
                  value=""
                />
                <div
                  class="chakra-input__right-element css-wp8zsj"
                >
                  <button
                    aria-label="Close"
                    class="css-sr8yzb"
                    type="button"
                  >
                    <svg
                      aria-hidden="true"
                      class="chakra-icon css-onkibi"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M.439,21.44a1.5,1.5,0,0,0,2.122,2.121L11.823,14.3a.25.25,0,0,1,.354,0l9.262,9.263a1.5,1.5,0,1,0,2.122-2.121L14.3,12.177a.25.25,0,0,1,0-.354l9.263-9.262A1.5,1.5,0,0,0,21.439.44L12.177,9.7a.25.25,0,0,1-.354,0L2.561.44A1.5,1.5,0,0,0,.439,2.561L9.7,11.823a.25.25,0,0,1,0,.354Z"
                        fill="currentColor"
                      />
                    </svg>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </form>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
